pysite/database.py

```diff
@@ -8,13 +9,16 @@
 from rethinkdb.ast import RqlMethodQuery, Table, UserError
 from rethinkdb.net import DefaultConnection
 
+from pysite.constants import DEBUG_MODE
+
 ALL_TABLES = {
     # table: primary_key
-
     "oauth_data": "id",
     "tags": "tag_name",
     "users": "user_id",
     "wiki": "slug",
+    "hiphoppers": "hophop_name",
```

Comment:
Shouldn't this be `hiphop_name`?

Reply:
oh.. yeah it should. :D 

It could probably just be `name`

Reply:
Well really I do think that tables should be sectioned off from other parts of the database - so like hh_users, hh_names, that kind of thing.

Also, this dict should be alphabetical, surely.